airsonic-main/src/main/webapp/WEB-INF/jsp/playlist.jsp

```diff
@@ -44,7 +45,7 @@
                     callback({data: songs});
                 },
                 stripeClasses: ["bgcolor2", "bgcolor1"],
-                columnDefs: [{ targets: "_all", orderable: false }],
+                columnDefs: [{ targets: "_all", orderable: true }],
```

Comment:
should this be orderable if edits are not allowed by the role?

Reply:
you are right, fixed